chainstate/storage/src/lib.rs

```diff
@@ -15,6 +15,7 @@
 
 //! Application-level interface for the persistent blockchain storage.
 
+pub mod db;
```

Comment:
why not `schema` for module name? looks more specific

Reply:
I called it `schema` initially but then it had the same name `storage_core::schema` and it is often useful to import both, making it awkward to work with. If you have an idea for a better name or module layout that does not collide with the generic `storage_core::schema`, I will be very happy to hear about alternatives.

Reply:
Also I thought the names of the database maps could be changed now that the schema is in its own submodule. For exmaple the `DBBlock` map is now referred to as `db::DBBlock` (after importing `chainstate_storage::db`). This could b made more concise by calling it just `db::Block` as it would no longer collide with `chain::Block`. Thoughts/comments?

Reply:
tbh I don't see a problem with both modules being `schema`. We do the same for `error` modules. As a solution for confusing `Schema` names one can be called `ChainstateSchema`.

> This could b made more concise by calling it just db::Block as it would no longer collide with chain::Block.

we can import via `use chainstate_storage::schema::Block as DBBlock`. Though imo this rename can be more confusing than helpful.

But maybe I don't see the full picture, so treat my comment as a suggestion

Reply:
> tbh I don't see a problem with both modules being `schema`. We do the same for `error` modules. As a solution for confusing `Schema` names one can be called `ChainstateSchema`.

Well at the moment I import both as modules, not the individual items in the modules, so importing both is a name clash:
```rust
use storage::schema;
use chainstate_storage::schema;  // conflicts with the above
```

I think it can be resolved by renaming it locally only in modules that use both, which may be a small improvement but still not ideal:
```rust
use storage::schema;
use chainstate_storage::schema as db;
```